package/samplers/nsgaii_with_initial_trials/nsgaii_with_initial_trials.py

```diff
@@ -0,0 +1,385 @@
+            Seed for random number generator.
+
+        constraints_func:
+            An optional function that computes the objective constraints. It must take a
+            :class:`~optuna.trial.FrozenTrial` and return the constraints. The return value must
+            be a sequence of :obj:`float` s. A value strictly larger than 0 means that a
+            constraints is violated. A value equal to or smaller than 0 is considered feasible.
+            If ``constraints_func`` returns more than one value for a trial, that trial is
+            considered feasible if and only if all values are equal to 0 or smaller.
+
+            The ``constraints_func`` will be evaluated after each successful trial.
+            The function won't be called when trials fail or they are pruned, but this behavior is
+            subject to change in the future releases.
+
+            The constraints are handled by the constrained domination. A trial x is said to
+            constrained-dominate a trial y, if any of the following conditions is true:
+
+            1. Trial x is feasible and trial y is not.
+            2. Trial x and y are both infeasible, but trial x has a smaller overall violation.
+            3. Trial x and y are feasible and trial x dominates trial y.
+
+            .. note::
+                Added in v2.5.0 as an experimental feature. The interface may change in newer
+                versions without prior notice. See
+                https://github.com/optuna/optuna/releases/tag/v2.5.0.
+
+        elite_population_selection_strategy:
+            The selection strategy for determining the individuals to survive from the current
+            population pool. Default to :obj:`None`.
+
+            .. note::
+                The arguments ``elite_population_selection_strategy`` was added in v3.3.0 as an
+                experimental feature. The interface may change in newer versions without prior
+                notice.
+                See https://github.com/optuna/optuna/releases/tag/v3.3.0.
+
+        child_generation_strategy:
+            The strategy for generating child parameters from parent trials. Defaults to
+            :obj:`None`.
+
+            .. note::
+                The arguments ``child_generation_strategy`` was added in v3.3.0 as an experimental
+                feature. The interface may change in newer versions without prior notice.
+                See https://github.com/optuna/optuna/releases/tag/v3.3.0.
+
+        after_trial_strategy:
+            A set of procedure to be conducted after each trial. Defaults to :obj:`None`.
+
+            .. note::
+                The arguments ``after_trial_strategy`` was added in v3.3.0 as an experimental
+                feature. The interface may change in newer versions without prior notice.
+                See https://github.com/optuna/optuna/releases/tag/v3.3.0.
+    """
+
+    def __init__(
+        self,
+        *,
+        n_init_trials: int = 50,
```

Comment:
[question] Where are you using this argument?
I cannot find this parameter in the original implementation as well.

Reply:
I created the argument thinking I would use it the first time, but forgot to delete it because I never ended up using it.
Correction.